compiler/fir/analysis-tests/testData/resolve/fromBuilder/enums.kt

```diff
@@ -23,7 +23,7 @@ enum class Planet(val m: Double, internal val r: Double) {
         }
     };
 
-    val g: Double = G * m / (r * r)
+    val g: Double = <!UNINITIALIZED_VARIABLE!>G<!> * m / (r * r)
```

Comment:
Why so? It's `const val` declared in companion object.

Reply:
Actually this is correct (FE 1.0 reports error here)

Imagine that `G` is marked with `@JvmStatic`

Reply:
Yes, you are right.